uint8 JOINT = 0
uint8 GIMBAL = 1

uint8 type
uint8 servo_id

int8 angle_sgn
float32 angle_scale
int16 zero_point_offset
